editor/editor_atlas_packer.cpp

```diff
@@ -59,7 +59,7 @@ void EditorAtlasPacker::chart_pack(Vector<Chart> &charts, int &r_width, int &r_h
 
 		Ref<BitMap> src_bitmap;
 		src_bitmap.instantiate();
-		src_bitmap->create(aabb.size / divide_by);
+		src_bitmap->create((Vector2(aabb.size) / divide_by).ceil());
```

Comment:
Now using ceil instead of treating the zero case special, can someone take a look at that, makes more sense to me than either truncation (which happens here) or rounding